contrib/google.golang.org/grpc/grpc.go

```diff
@@ -31,11 +31,13 @@ func startSpanFromContext(ctx context.Context, method, operation, service string
 	return tracer.StartSpanFromContext(ctx, operation, opts...)
 }
 
-// withStreamError returns a tracer.WithError finish option, disregarding OK, EOF and Canceled errors.
-func withStreamError(err error) tracer.FinishOption {
+// finishWithStreamError applies finish option and a tag with gRPC status code, disregarding OK, EOF and Canceled errors.
+func finishWithStreamError(span ddtrace.Span, err error) {
 	errcode := status.Code(err)
 	if err == io.EOF || errcode == codes.Canceled || errcode == codes.OK || err == context.Canceled {
 		err = nil
+		errcode = codes.OK
```

Comment:
We shouldn't overwrite the code, no matter what it is. The user should always see the real code.

Reply:
OK. probably at least we shoud ignore io.EOF because always returns Unknow for last rpc.